lib/jekyll/commands/serve/servlet.rb

```diff
@@ -3,6 +3,128 @@
 module Jekyll
   module Commands
     class Serve
+      # This class is used to determine if the Servlet should modify a served file
+      # to insert the LiveReload script tags
+      class SkipAnalyzer
+        BAD_USER_AGENTS = [%r!MSIE!].freeze
+
+        def self.skip_processing?(req, res, options)
```

Comment:
I'd appreciate seeing some more expressive variable names here. I can parse that `req` means `request` and `res` means `response` but I think it's better if we just use those variable names rather than having people who come along later reading the code to parse them too.